src/pages/communtiyapplication.js

```diff
@@ -1,12 +1,18 @@
+import React from 'react';
 import { Footer } from '../components/common/footer';
 import { Header } from '../components/common/header';
 import { CommunitiesApplication } from '../components/pages/landing/communitiesApplication';
 
 export const CommunityApplicationPage = ({ isSignedIn }) => {
+  const ref = React.useRef();
+
+  React.useEffect(() => {
+    document.body.scrollTop = document.documentElement.scrollTop = 0;
```

Comment:
window.scrollTo(0, 0)
this should work
I don't think we need to assign ref for this